syntax/commands.js

```diff
@@ -1345,4 +1352,341 @@ module.exports = [
+    spec: "distance",
+    shape: "reporter",
+    category: "ev3",
+  },
+  {
+    id: "ev3.getBrightness",
+    spec: "brightness",
+    shape: "reporter",
+    category: "ev3",
+  },
+  {
+    id: "ev3.beepNote",
+    spec: "beep note %1 for %2 secs",
+    inputs: ["%d.note", "%n"], // we can use %d.note here
+    shape: "stack",
+    category: "ev3",
+  },
+  {
+    id: "gdxfor.whenGesture",
+    spec: "when %1",
+    inputs: ["%m"],
+    shape: "hat",
+    category: "gdxfor",
+  },
+  {
+    id: "gdxfor.whenForcePushedOrPulled",
+    spec: "when force sensor %1",
+    inputs: ["%m"],
+    shape: "hat",
+    category: "gdxfor",
+  },
+  {
+    id: "gdxfor.getForce",
+    spec: "force",
+    shape: "reporter",
+    category: "gdxfor",
+  },
+  {
+    id: "gdxfor.whenTilted",
+    spec: "when tilted %1",
+    inputs: ["%m"],
+    shape: "hat",
+    category: "gdxfor",
+  },
+  {
+    id: "gdxfor.isTilted",
+    spec: "tilted %1?",
+    inputs: ["%m"],
+    shape: "boolean",
+    category: "gdxfor",
+  },
+  {
+    id: "gdxfor.getTilt",
+    spec: "tilt angle %1",
+    inputs: ["%m"],
+    shape: "reporter",
+    category: "gdxfor",
+  },
+  {
+    id: "gdxfor.isFreeFalling",
+    spec: "falling?",
+    shape: "boolean",
+    category: "gdxfor",
+  },
+  {
+    id: "gdxfor.getSpin",
+    spec: "spin speed %1",
+    inputs: ["%m"],
+    shape: "reporter",
+    category: "gdxfor",
+  },
+  {
+    id: "gdxfor.getAcceleration",
+    spec: "acceleration %1",
+    inputs: ["%m"],
+    shape: "reporter",
+    category: "gdxfor",
+  },
+  {
+    id: "microbit.whenButtonPressed",
+    spec: "when %1 button pressed",
+    inputs: ["%m"],
+    shape: "hat",
+    category: "microbit",
+  },
+  {
+    id: "microbit.isButtonPressed",
+    spec: "%1 button presses?",
+    inputs: ["%m"],
+    shape: "boolean",
+    category: "microbit",
+  },
+  {
+    id: "microbit.whenGesture",
+    spec: "when %1",
+    inputs: ["%m"],
+    shape: "hat",
+    category: "microbit",
+  },
+  {
+    id: "microbit.displaySymbol",
+    spec: "display %1",
+    inputs: ["%m"], // TODO add matrix support
+    shape: "stack",
+    category: "microbit",
+  },
+  {
+    id: "microbit.displayText",
+    spec: "display text %1",
+    inputs: ["%s"],
+    shape: "stack",
+    category: "microbit",
+  },
+  {
+    id: "microbit.clearDisplay",
+    spec: "clear display",
+    shape: "stack",
+    category: "microbit",
+  },
+  {
+    id: "microbit.whenTilted",
+    spec: "when tilted %1",
+    inputs: ["%m"],
+    shape: "hat",
+    category: "microbit",
+  },
+  {
+    id: "microbit.isTilted",
+    spec: "tilted %1?",
+    inputs: ["%m"],
+    shape: "boolean",
+    category: "microbit",
+  },
+  {
+    id: "microbit.tiltAngle",
+    spec: "tilt angle %1",
+    inputs: ["%m"],
+    shape: "reporter",
+    category: "microbit",
+  },
+  {
+    id: "microbit.whenPinConnected",
+    spec: "when pin %1 connected",
+    inputs: ["%m"],
+    shape: "hat",
+    category: "microbit",
+  },
```

Comment:
There's a set of text files under `tests/` which list out the blocks in the same order they appear in the Scratch interface. I like to use those to check at a glance that everything looks the same as the Scratch interface.

Would you be up for providing something similar for all the extensions? There's a lot of blocks here, it would be cool to check we haven't made any typos!